src/etc/inc/util.inc

```diff
@@ -1142,6 +1147,11 @@ function is_portoralias($port) {
 	}
 }
 
+/* returns true if $port is a valid TCP/UDP port number or range ("<port>:<port>") or an alias thereof */
+function is_portorrangeoralias($port) {
+	return (is_portoralias($port) || is_portrange($port));
```

Comment:
I guess you meant is_portorrange() instead of is_portrange() here, no?
I'd also suggest to reverse this clause since is_portorrange() is cheaper than is_portoralias()